Game/src/main/MainApp.java

```diff
@@ -20,14 +21,52 @@ public static void main(String[] args) {
         RenderEngine engine = RenderEngine.init();
         SceneLoader loader = SceneLoaderFactory.createSceneLoader();
         Scene scene = loader.loadScene(new MyFile(LoaderSettings.RES_FOLDER, "Socuwan Scene"));
-
+        /*
         MyFile textureFile = new MyFile(LoaderSettings.RES_FOLDER
                                         + "/Socuwan Scene/Gui Texture/LxpnJ.png");
         scene.addGUITexture(new GUITexture(Texture.newTexture(textureFile).create().textureId,
                                            new Vector2f(0.5f, 0.5f), new Vector2f(0.05f,0.05f)));
-
+        */
         engine.renderEnvironmentMap(scene.getEnvironmentMap(), scene, new Vector3f(0, 2, 0));
 
+        Mouse.addMouseListener(new MouseListener() {
```

Comment:
Jeg tænker at man nok hellere vil have en klasse med noget funktionalitet der implementerede MouseListener eller KeyboardListener. Kunne vi lave noget fancy med at enginen leder efter dem?
Sådan at vi ved at alle der implementere MouseListeners, som måske allerede er scene entities eller noget i den stil. Eller bliver det for magisk?